### DISCUSSION TITLE
Can I avoid launching a new node engine every time I save a chart?

### DISCUSSION THREAD

**Discussion by AdamCiuris** [vega/altair#2967] [Q&A]
I'm running the saver in a loop for thousands of files. It looks like it makes a new backend for each one which is slowing things down a lot. Any edits I could make to avoid this? I don't think there is an option for it.

**Comment by joelostblom** [comment]
The [latest release candidate](https://github.com/altair-viz/altair/issues/2937) of altair uses [vl-convert](https://github.com/vega/vl-convert) for saving, which you could also use manually on older version of altair. Not sure how it compares in speed, but it might be worth a try?
